modules/apps/commerce/commerce-shipping-engine-fixed-web/src/main/java/com/liferay/commerce/shipping/engine/fixed/web/internal/FixedCommerceShippingEngine.java

```diff
@@ -160,7 +160,7 @@ public String getName(Locale locale) {
 			BigDecimal amount = commerceShippingFixedOption.getAmount();
 
 			CommerceChannel commerceChannel =
-				_commerceChannelService.getCommerceChannelByOrderGroupId(
+				_commerceChannelLocalService.getCommerceChannelByOrderGroupId(
```

Comment:
@alecsloan why are we changing this? this was working before so the root cause should be something else

Reply:
Hey @riccardo-alberti the only possible thing I see that it could have been caused by was COMMERCE-4982, however I don't see a need to use remote services here regardless since we're just using the channel to get a currency.

Reply:
@alecsloan i see that this logic for getting the channel and converting the currency was added after 7.3 that's why it was working there. Anyway as long as we are not introducing security issues by using the local service it's fine by me